compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -185,7 +204,9 @@ impl CoverageMapGenerator {
                 end_col,
             ));
         }
+    }
 
+    {
```

Comment:
(These extra braces are just there to avoid re-indenting the same code multiple times; they get removed by other patches by the end of this PR.)